src/ir.h

```diff
@@ -501,11 +528,20 @@ class SimdShuffleOpExpr : public ExprMixin<ExprType::SimdShuffleOp> {
 
 template <ExprType TypeEnum>
 class VarExpr : public ExprMixin<TypeEnum> {
- public:
-  VarExpr(const Var& var, const Location& loc = Location())
+  public:
+    VarExpr(const Var& var, const Location& loc = Location())
```

Comment:
I think the old indentation is what clang-format is setup to do for this project.